app/feature/feature-choose-tier/src/main/kotlin/com/hedvig/android/feature/change/tier/ui/stepcustomize/SelectTierDestination.kt

```diff
@@ -197,6 +206,8 @@ private fun SelectTierScreen(
   onChooseTierClick: () -> Unit,
   onChooseDeductibleInDialogClick: (quote: TierDeductibleQuote) -> Unit,
   onChooseTierInDialogClick: (tier: Tier) -> Unit,
+  onSetDeductibleBackToPreviouslyChosen: () -> Unit,
+  onSetTierBackToPreviouslyChosen: () -> Unit,
```

Comment:
So I think here, changes in this screen  (and its ViewModel) are mostly what needs to be reviewed again just a little bit. Otherwise it's just small fixes according to the previous comments.
What I'm trying to achieve here, is that we would be able to press cancel (or outside of the dialog), and the choice locally made in dialog would be dismissed, and we would go back to what was previously chosen. That's why I'm storing two choices - "global" and "dialog local".
I tried not to pass local choice to the viewModel at all, but it behaved strangely sometimes then, so everything goes through vm now.